### ISSUE TITLE
Audit.WebApi is not compatible with ASP.NET Core 2.1

### ISSUE THREAD

**Issue by adrianiftode** [thepirat000/Audit.NET#284]
**Describe the bug**
We are having some web apps that target netcoreapp2.1 and want to install the Audit.WebApi package, but we can't as we get an error. 2.1 is LTS is we won't going to change them too soon.

**To Reproduce**

Given a web apps that target netcoreapp2.1.

When I want to install the Audit.WebApi package

Then I receive an error with the following description
```
Install-Package : NU1107: Version conflict detected for Microsoft.AspNetCore.Antiforgery. Install/reference Microsoft.AspNetCore.Antiforgery 2.2.0 
directly to project Test2.1 to resolve this issue. 
 Test2.1 -> Audit.WebApi 15.0.5 -> Microsoft.AspNetCore.Mvc 2.2.0 -> Microsoft.AspNetCore.Mvc.ViewFeatures 2.2.0 -> 
Microsoft.AspNetCore.Antiforgery (>= 2.2.0) 
 Test2.1 -> Microsoft.AspNetCore.App 2.1.1 -> Microsoft.AspNetCore.Antiforgery (>= 2.1.1 && < 2.2.0).
At line:1 char:1
+ Install-Package Audit.WebApi
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
 
Install-Package : Package restore failed. Rolling back package changes for 'Test2.1'.
At line:1 char:1
+ Install-Package Audit.WebApi
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
```

**Expected behavior**
The package to be installed

**Libraries (specify the Audit.NET extensions being used including version):**
 - Audit.WebApi: 15.0.5

**Additional context**
One possible fix is to change from
```
  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.2.0" />
  </ItemGroup>  
 ```
to
```
  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.1.0" />
  </ItemGroup>  
```

14.7.0 is a version that works with ASP.NET Core 2.1

**Comment by thepirat000** [comment]
Could you share a minimal .csproj file that reproduces the error?

**Comment by adrianiftode** [comment]
```
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>netcoreapp2.1</TargetFramework>
  </PropertyGroup>
  
  <ItemGroup>
    <PackageReference Include="Audit.WebApi" Version="15.0.5" />
    <PackageReference Include="Microsoft.AspNetCore.App" />
  </ItemGroup>

</Project>

```
[Audit.NET#284.csproj.zip](https://github.com/thepirat000/Audit.NET/files/4464902/Audit.NET.284.csproj.zip)
building this you'll get

```
Microsoft (R) Build Engine version 16.5.0+d4cbfca49 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

D:\adi\opensource\Audit.NET#284\Audit.NET#284.csproj : error NU1107: Version conflict detected for Microsoft.AspNetCore.Antiforgery. Install/reference Microsoft.AspNetCore.Antiforgery 2.2.0 directly to project Audit.NET#284 to resolve this issue.
D:\adi\opensource\Audit.NET#284\Audit.NET#284.csproj : error NU1107:  Audit.NET#284 -> Audit.WebApi 15.0.5 -> Microsoft.AspNetCore.Mvc 2.2.0 -> Microsoft.AspNetCore.Mvc.ViewFeatures 2.2.0 -> Microsoft.AspNetCore.Antiforgery (>= 2.2.0)
D:\adi\opensource\Audit.NET#284\Audit.NET#284.csproj : error NU1107:  Audit.NET#284 -> Microsoft.AspNetCore.App 2.1.1 -> Microsoft.AspNetCore.Antiforgery (>= 2.1.1 && < 2.2.0).
  Restore failed in 955.17 ms for D:\adi\opensource\Audit.NET#284\Audit.NET#284.csproj.

Build FAILED.

D:\adi\opensource\Audit.NET#284\Audit.NET#284.csproj : error NU1107: Version conflict detected for Microsoft.AspNetCore.Antiforgery. Install/reference Microsoft.AspNetCore.Antiforgery 2.2.0 directly to project Audit.NET#284 to resolve this issue.
D:\adi\opensource\Audit.NET#284\Audit.NET#284.csproj : error NU1107:  Audit.NET#284 -> Audit.WebApi 15.0.5 -> Microsoft.AspNetCore.Mvc 2.2.0 -> Microsoft.AspNetCore.Mvc.ViewFeatures 2.2.0 -> Microsoft.AspNetCore.Antiforgery (>= 2.2.0)
D:\adi\opensource\Audit.NET#284\Audit.NET#284.csproj : error NU1107:  Audit.NET#284 -> Microsoft.AspNetCore.App 2.1.1 -> Microsoft.AspNetCore.Antiforgery (>= 2.1.1 && < 2.2.0).
    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:01.60
```


Another way to reproduce
```
dotnet new web --framework netcoreapp2.1
dotnet add package Audit.WebApi
```
which will output
```
D:\adi\opensource\Audit.NET#284>dotnet add package Audit.WebApi
  Writing C:\Users\Adrian Iftode\AppData\Local\Temp\tmpDF6E.tmp
info : Adding PackageReference for package 'Audit.WebApi' into project 'D:\adi\opensource\Audit.NET#284\Audit.NET#284.csproj'.
info : Restoring packages for D:\adi\opensource\Audit.NET#284\Audit.NET#284.csproj...
info :   CACHE https://api.nuget.org/v3-flatcontainer/audit.webapi/index.json
error: Version conflict detected for Microsoft.AspNetCore.Antiforgery. Install/reference Microsoft.AspNetCore.Antiforgery 2.2.0 directly to project Audit.NET#284 to resolve this issue.
error:  Audit.NET#284 -> Audit.WebApi 15.0.5 -> Microsoft.AspNetCore.Mvc 2.2.0 -> Microsoft.AspNetCore.Mvc.ViewFeatures 2.2.0 -> Microsoft.AspNetCore.Antiforgery (>= 2.2.0)
error:  Audit.NET#284 -> Microsoft.AspNetCore.App 2.1.1 -> Microsoft.AspNetCore.Antiforgery (>= 2.1.1 && < 2.2.0).
info : Package 'Audit.WebApi' is compatible with all the specified frameworks in project 'D:\adi\opensource\Audit.NET#284\Audit.NET#284.csproj'.
info : PackageReference for package 'Audit.WebApi' version '15.0.5' updated in file 'D:\adi\opensource\Audit.NET#284\Audit.NET#284.csproj'.
info : Committing restore...
info : Writing assets file to disk. Path: D:\adi\opensource\Audit.NET#284\obj\project.assets.json
log  : Restore failed in 1.13 sec for D:\adi\opensource\Audit.NET#284\Audit.NET#284.csproj.
```

**Comment by thepirat000** [comment]
Fixed on version 15.1.0, please updgrade your references and re-test

**Comment by adrianiftode** [comment]
Looks great, thank you!
